client/src/components/Avatar.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import Profile from '../assets/profile/Tina.jpeg';
+import cameraIcon from '../assets/camera.svg';
+import PropTypes from 'prop-types';
+
+const StyledProfileImg = styled.img`
+  height: 100px;
+  width: 100px;
```

Comment:
That's true, but I'm gonna put it on my refactoring list because I didn't get it fixed at the moment🙈 If you know a better solution I will be glad for you help 🤗